yarn.lock

```diff
@@ -371,6 +371,11 @@ async@^1.5.2:
   resolved "https://registry.yarnpkg.com/async/-/async-1.5.2.tgz#ec6a61ae56480c0c3cb241c95618e20892f9672a"
   integrity sha1-7GphrlZIDAw8skHJVhjiCJL5Zyo=
 
+attr-accept@^2.2.1:
```

Comment:
Do we need these yarn.lock changes?

Reply:
To be honest, i'm not sure when I should be committing yarn.lock changes and when they are extraneous

Reply:
You should never have to change yarn.lock unless you're adding/removing something to/from packages.json

Reply:
(to revert this file, you can run `git checkout master -- yarn.lock` then add + commit + push the changes)